src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
remove the import React from "react" because you are not using it anywhere in your code!